Formula/qfs.rb

```diff
@@ -0,0 +1,101 @@
+class Qfs < Formula
+  desc "High-performance, fault-tolerant, distributed file system and its tools"
+  homepage "https://github.com/quantcast/qfs"
+  url "https://github.com/quantcast/qfs.git"
+  version "1.1.4-1"
+  head "https://github.com/quantcast/qfs.git"
+
+  bottle :unneeded
+
+  depends_on "boost"
+  depends_on "cmake" => :build
+  depends_on "maven" => :build
+  depends_on "openssl" => :build
+  depends_on "wget"
+
+  def install
+    ENV.java_cache
+
+    opoo "Building qfs. This may take some time..."
+    system "make", "build", "hadoop-jars", "BUILD_TYPE=release"
+
+    bin.install "build/release/bin/chunkscrubber"
```

Comment:
Is there no `make install` we can use? If not, can you go `prefix.install "build/release/bin"`?